packages/sdk/src/server/index.ts

```
@@ -107,6 +107,19 @@
 
+/**
+ * Parameters for the retrieval of an account from the Connect API
+ */
+export type MakeProxyRequestOpts = {
+  /**
+   * Whether to retrieve the account's credentials or not.
+   */
+  method: string;
+  headers?: Record<string, string>;
+  //headers: Record<string, string>;
+  body?: any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)